app/assets/stylesheets/common/components/keyboard_shortcuts.scss

```
@@ -105,6 +105,11 @@
       margin-left: auto;
     }
 
+    span.delimiter-newline {
```

No need for the extra tag specificity I think
```suggestion
    .delimiter-newline {
```

---

Good point, I needed it to override the `:first-child`/`:last-child` margin rules while I was experimenting with layouts, but not any more. 🙂